addon/components/influencer-network-modal.js

```diff
@@ -1,10 +1,36 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
 import { GLOBAL_SUPPORT_LINK } from '@upfluence/ember-upf-utils/resources/helpdesk-links';
+import { inject as service } from '@ember/service';
 
 const INFLUENCER_NETWORK_MODAL_COOKIE = 'upf_disable_influencer_modal';
 
 export default Component.extend({
+  currentUser: service(),
+  user: null,
+
+  init() {
+    this._super();
+    this.currentUser.fetch().then((user) => {
+      this.set('user', user);
+    });
+  },
+
+  displayInfluencerNetworkModal: computed(
+    'hasDisabledInfluencerNetworkModal',
+    'hideInfluencerNetworkModal',
+    'user.companies.{firstObject.billing_format,length}',
+    function () {
+      return (
+        !this.hasDisabledInfluencerNetworkModal &&
+        !this.hideInfluencerNetworkModal &&
```

Comment:
I didn't see `hideInfluencerNetworkModal` in this file, are you sure it is not `this.args`?

Reply:
you're rigth that doesn't did exception when I test it so I did see it when I put in the controller instead of the template

Reply:
Up! when i do `this.hideInfluencerNetworkModal` in console.log it return me `true` but when I did `this.args.hideInfluencerNetworkModal` in console.log() that throw an exception that say 'Cannot reading property...'
But I don't know where it comes from.

Reply:
Last up, hideInfluencerNetworkModal is a param pass to the component but this.args doesn't work, maybe because this is old syntax...